packages/runtime/container-runtime/src/summaryCollection.ts

```diff
@@ -273,15 +284,15 @@ export class SummaryCollection {
     private handleOp(op: ISequencedDocumentMessage) {
         switch (op.type) {
             case MessageType.Summarize: {
-                this.handleSummaryOp(op as ISummaryOpMessage);
+                this.handleSummaryOp(op as ISummaryOpMessage, this.opActions);
```

Comment:
Just curious, why pass `this.opActions` through, when it's still accessible within `this.handleSummaryOp` etc.?

Reply:
no good reason. i'll update it